### DISCUSSION TITLE
Some Quesitons about the wiki.js (Permission and Redirection)

### DISCUSSION THREAD

**Discussion by Skyer19** [requarks/wiki#5249] [Help / Questions]
Hi, I have some quesitons about the wiki.js.

First, I used the domain address instead of the ip address, and I want the the domain to redirect to the home page. However, when I input the domain address, the domain address will redirect to the login page rather than the home page. I have tried many methods but I did not fing any appropriate solutions?

Second, I want someone to visit all the pages in my wiki without login. I have a Guest group, and I have given all read rules, but when I visit my page, it will redirct to the login page or say "No Premission".

I am now very confused and I hope if someone could help me solve them. Thanks.

OS: CentOS 7.9 64bit
Wiki.js version: 2.5.279
Database engine: PostgreSQL 13.6

**Comment by NGPixel** [comment]
Post the permissions and page rules you created for the Guests group.

**Reply by Skyer19** [reply]
Here are them! Please check them, thanks.
![截屏2022-05-03 上午9 06 27](https://user-images.githubusercontent.com/58378454/166390860-f3d7ddea-c1c2-4590-8ada-bcc115013965.png)
![截屏2022-05-03 上午9 06 10](https://user-images.githubusercontent.com/58378454/166390856-eb4d5c7e-8b94-49de-abf2-cc4bc1763a7c.png)

**Reply by NGPixel** [reply]
That's because you have no rule for path `/`, which is the page guests land on. You need to give read permission at path `/`.

**Comment by Skyer19** [comment]
Do you mean like this? But when I enter my domain address, it also says  "No Premission" and ask me to login it. :(
![截屏2022-05-03 下午2 41 38](https://user-images.githubusercontent.com/58378454/166413861-e76128f4-9cd3-453c-a80b-2035134d2f15.png)
![截屏2022-05-03 下午2 44 01](https://user-images.githubusercontent.com/58378454/166413864-cf2c2bd0-5273-4ba0-a6b2-2cfebf43828d.png)
